docs/index.rst

```diff
@@ -24,10 +26,13 @@ The system utilizes GPS data to determine real-time location information, such a
    :caption: Contents:
 
 
-   GUI<gui/index>
-   Software<software/index>
-   User Guide<how_to>
-   Comp Manager<component_manager_api>
+   Getting started<getting_started>
+   Installation<how_to_install>
+   User Guide<usage>
+   Developer Guide<dev_practices>
+   GUI API<gui/index>
+   Comp Manager API<component_manager_api>
+   Additional API<software/index>
```

Comment:
This is amazing, just remove the starting the network part if we do not need it.

Reply:
@USilere Please help me understand where you are seeing the network part. I am struggling to see it.

Reply:
Additional API index

Reply:
Please see screenshot
![Uploading Screenshot from 2024-07-25 09-07-25.png…]()

Reply:
I will remove it , Thanks.